custom_json_diff/custom_diff_classes.py

```diff
@@ -352,7 +353,6 @@ def get_cdxgen_excludes(includes: List[str], comp_only: bool, allow_new_versions
                 'metadata.tools.components.[].bom-ref': 'metadata.tools.components.[].bom-ref',
                 'properties': 'components.[].properties', 'evidence': 'components.[].evidence',
                 'licenses': 'components.[].licenses', 'hashes': 'components.[].hashes',
-                'externalReferences': 'components.[].externalReferences',
```

Comment:
@prabhu I prefer to keep this to avoid any someone not noticing it is case-sensitive.

Reply:
@prabhu No, the other one is externalreferences all lowercase 😉